src/test/python/MCSVerification.py

```diff
@@ -63,38 +64,32 @@ def _run_scispark_implementation():
     Purpose: To run GTG runner for SciSpark results to compare as a spark-submit task
     Assumptions: spark-submit is available on the path. scala-2.11 is used.
     '''
-    runSpark = False
-
     print 'Checking on SciSpark implementation data' 
     of.write('\n Checking on SciSpark implementation data')
 
     try:
-        if not os.path.exists(workingDir + '/scisparkGTG'):
+        if os.path.exists(workingDir + '/scisparkGTG'):
+            shutil.rmtree(workingDir + '/scisparkGTG')
+        else:
             os.mkdir(workingDir + '/scisparkGTG')
-            runSpark = True
-        if not os.path.exists(workingDir + '/scisparkGTG/textFiles'):
             os.mkdir(workingDir + '/scisparkGTG/textFiles')
-            runSpark = True
-        if not os.path.exists(workingDir + '/scisparkGTG/MERGnetcdfCEs'):
             os.mkdir(workingDir + '/scisparkGTG/MERGnetcdfCEs')
-            runSpark = True
-
-        if runSpark:
-            os.chdir(workingDir + '/../../../../')
-
-            sparkSubmitStr = 'spark-submit target/scala-2.11/SciSpark.jar'
-            subprocess.call(sparkSubmitStr, shell=True)
-            cpTextFilesStr = 'mv MCCEdges.txt ' + workingDir + '/scisparkGTG/textFiles'
-            subprocess.call(cpTextFilesStr, shell=True)
-            cpTextFilesStr = 'mv MCCNodesLines.json ' + workingDir + '/scisparkGTG/textFiles'
-            subprocess.call(cpTextFilesStr, shell=True)
-            cpNetcdfsStr = 'mv /tmp/*.nc ' + workingDir + '/scisparkGTG/MERGnetcdfCEs'
-            subprocess.call(cpNetcdfsStr, shell=True)
-
-            print 'SciSpark implementation successfully run. Data can be found at %s' \
-                %workingDir + '/scisparkGTG'
-            of.write('\nSciSpark implementation successfully run. Data can be found at %s' \
-                %workingDir + '/scisparkGTG')
+
+        os.chdir(workingDir + '/../../../../')
+
+        sparkSubmitStr = 'spark-submit target/scala-2.11/SciSpark.jar'
+        subprocess.call(sparkSubmitStr, shell=True)
+        cpTextFilesStr = 'mv MCCEdges.txt ' + workingDir + '/scisparkGTG/textFiles'
```

Comment:
@kwhitehall 
Could we do cp MCCEdges.txt rather than move?
The same goes for the other mv commands.

That way we preserve the copies in the base directory,